src/Redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const ADD_BOOK = 'addBook';
+const REMOVE_BOOK = 'removeBook';
```

Comment:
- [ ] According to the requirements, kindly ensure that the values of these action types follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux). It should look something like this:
```const ADD_BOOK = bookstore/books/ADD_BOOK```
```const REMOVE_BOOK = bookstore/books/REMOVE_BOOK```